indexer/editable_map_object.cpp

```diff
@@ -193,7 +85,7 @@ vector<MapObject::MetadataID> EditableMapObject::GetEditableProperties() const
   return props;
 }
 
-NamesDataSource EditableMapObject::GetNamesDataSource(bool needFakes /* = true */)
+NamesDataSource EditableMapObject::GetNamesDataSource(bool needFakes /* = false */)
```

Comment:
Can needFakes parameter be removed?

Reply:
@kirylkaveryn The value of `needFakes` is ignored in the core, and you can also call the function with out a parameter. So you can safely remove all `needFakes` related code from iOS